Demo/Common/IntegrationExampleComponents.swift

```diff
@@ -128,8 +128,12 @@ extension IntegrationExample {
         }
     }
 
-    private func handle(_ action: Action) {
-        actionComponent.handle(action)
+    internal func handle(_ action: Action) {
+        if let actionComponent = currentComponent as? ActionHandlingComponent {
+            actionComponent.handle(action)
+        } else {
+            actionComponent.handle(action)
```

Comment:
I don't think this is good idea! 
Previously we had 2 separate flows per file: one for drop-In, one for components.

Now we have:
*  super confusing `if X { A } else { A }`;
* drop-in and component flows are mixed in one file;
* it is unobvious how to handle action for Drop-In flow.

Reply:
I think this is why the Voucher component was looking completely off, it was presented modally instead of through DropIn, somehow even though the two functions `handle(_ action:)` are private, they are still conflicting, because its still the same class.